internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -205,7 +201,7 @@ func (r *ISBServiceRolloutReconciler) reconcile(ctx context.Context, isbServiceR
 				return ctrl.Result{}, err
 			}
 			// Get the nfcRollout live resource
```

Comment:
looks like there was already a typo in the comment above "nfcRollout" - can we fix it here?